docs/sources/administration/api-keys/index.md

```diff
@@ -106,9 +106,9 @@ To migrate a single API key to a service account, complete the following steps:
 1. Find the API Key you want to migrate.
 1. Click **Migrate to service account**.
 
-### Migrate API keys to Grafana service accounts using the API
+### Migrate API keys to Grafana service accounts for API calls
 
-This section shows you how to migrate API keys to Grafana service accounts using the Grafana API.
+This section shows you how to migrate API keys to Grafana service accounts specifically for the Grafana API. For references see: [Grafana Service Accounts for the Grafana API](https://grafana.com/docs/grafana/latest/developers/http_api/serviceaccount/).
```

Comment:
This URL should redirect to the Service Account entry for the same documentation version you're in, not the latest.

I think you can achieve the same with a relative reference. Try the following code suggestion.

```suggestion
This section shows you how to migrate API keys to Grafana service accounts specifically for the Grafana API. For references see: [Grafana Service Accounts for the Grafana API]
({{< relref "..//service-accounts" >}})(https://grafana.com/docs/grafana/latest/developers/http_api/serviceaccount/).
```

Reply:
is this relref working?

Reply:
wanted to point to the http documentation. il update